### PR TITLE
Remove Discord discriminators and use handles in `Community Discord servers`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] ~*(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)~

### CODE REVIEW

**Comment by cl8n** [comment]
not sure if this really matters but I feel like discord tags would be more recognizable as such if they included the `@` prefix

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/de.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ Viele Medaillenjäger arbeiten zusammen, um Lösungen für jede neue Hush-Hush-M
 
 | Rolle | Benutzer (Discord-ID) |
 | :-- | :-- |
-| Server-Eigentümer | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151) |
-| Server-Admin | ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) (Hubz#6283, Tanza#8733), ::{ flag=DE }:: [mulraf](https://osu.ppy.sh/users/1309242) (mulraf#1087) |
-| Community-Manager & Server-Admin | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151), ::{ flag=GB }:: [chromb](https://osu.ppy.sh/users/10238680) (chromb#4528), ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) (Remyria#0001) |
-| Server-Moderator | ::{ flag=US }:: [bentokage](https://osu.ppy.sh/users/13175102) (bento#6969) |
+| Server-Eigentümer | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (@megamix_craft) |
+| Server-Admin | ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) (@tanza3d, @tanza.n), ::{ flag=DE }:: [mulraf](https://osu.ppy.sh/users/1309242) (@mulraf) |
```
Comment (Tanza3D) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/de.md:36]: could you exclude the second handle here for me? it's my personal account, thanks (likewise on the other languages)

Comment (agatemosu) [reply wiki/Community/Discord_servers/osu!_Medal_Hunters/de.md:36]: I thought the same

**Comment by Protastic101** [comment]
https://github.com/ppy/osu-wiki/pull/9812 No need to worry about updating the o!m mapping and modding hub article as I've got that covered here since we added some new moderators.

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
can do separate pr for osu university if you wanted to work on that one too. just wanted to get this in while im available

**Pull request merged**